File: src/StarAPI.java
package net.glitchtechs.starapi;

import java.util.logging.Level;
import org.bukkit.plugin.java.JavaPlugin;

public class StarAPI extends JavaPlugin {
    public void onEnable() {
        this.getLogger().log(Level.WARNING, "");
        this.getLogger().log(Level.WARNING, "StarAPI is now running on your server any plugin that requires this to run is now running!");
        this.getLogger().log(Level.WARNING, "");
    }
}

File: Kotlin/src/StarAPI.kt
package net.glitchtechs.starapi

import org.bukkit.plugin.java.JavaPlugin
import java.util.logging.Level

class StarAPI : JavaPlugin() {

    override fun onEnable() {
        logger.log(Level.WARNING, "")
        logger.log(Level.WARNING, "StarAPI is now running on your server. Any plugin that requires this to run is now running!")
        logger.log(Level.WARNING, "")
    }
}
